docs/word/word-add-ins-troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Troubleshooting Word add-ins
+description: Learn how to troubleshoot development errors in Word add-ins.
+ms.date: 02/01/2023
+ms.localizationpriority: medium
+---
+
+# Troubleshooting Word add-ins
```

Comment:
```suggestion
# Troubleshoot Word add-ins
```

Reply:
Considering what people might enter into search, there is a strong argument to leave the H1 as is. Up to you!